app/views/students/sped_referral.pdf.erb

```diff
@@ -138,6 +138,10 @@ ul, li {
     <% end %>
   </table>
 </p>
+<p>
+  <h4>Attendance graph (current and prior school year):</h4>
+  <div id="attendance_container"></div>
```

Comment:
We don't have good tooling for this now, but CSS classes are all hyphen case, so let's keep them consistent at least in the same file.